server/test/controllers/ti/TrustedIntermediaryControllerTest.java

```diff
@@ -0,0 +1,106 @@
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(
+            Helpers.fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "first",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "last",
+                        "emailAddress",
+                        "sample@fake.com")));
+    TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
+    Result result = tiController.addApplicant(group.id, requestBuilder.build());
+    assertThat(result.flash().get("error")).isEqualTo("providedDateOfBirth cannot be null");
+  }
+
+  @Test
+  public void addApplicantTestWithAllInformation() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(
+            Helpers.fakeRequest()
+                .bodyForm(
+                    ImmutableMap.of(
+                        "firstName",
+                        "first",
+                        "middleName",
+                        "middle",
+                        "lastName",
+                        "last",
+                        "emailAddress",
+                        "sample2@fake.com",
+                        "dob",
+                        "2022-07-18")));
+    TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
+    Result result = tiController.addApplicant(group.id + 1, requestBuilder.build());
+    assertThat(result.status()).isEqualTo(SEE_OTHER);
+    Optional<Applicant> testApplicant =
+        repo.lookupApplicantByEmail("sample2@fake.com").toCompletableFuture().join();
+    assertThat(testApplicant.get().getApplicantData().getDateOfBirth().get().toString())
+        .isEqualTo("2022-07-18");
+  }
+
+  @Test
+  public void testUpdateDOBFunctionWithExistingDob() {
+    Http.RequestBuilder requestBuilder =
+        addCSRFToken(Helpers.fakeRequest().bodyForm(ImmutableMap.of("dob", "2022-10-05")));
+    TrustedIntermediaryGroup group = repo.listTrustedIntermediaryGroups().get(0);
+    AddApplicantToTrustedIntermediaryGroupForm form =
+        new AddApplicantToTrustedIntermediaryGroupForm();
+    form.setEmailAddress("sample3@example.com");
+    form.setFirstName("foo");
+    form.setLastName("bar");
+    form.setDob("2022-07-10");
+    repo.createNewApplicantForTrustedIntermediaryGroup(form, group);
+    Optional<Applicant> applicant =
+        repo.lookupApplicantByEmail("sample3@example.com").toCompletableFuture().join();
+    Result result =
+        tiController.updateDateOfBirth(group.id, applicant.get().id, requestBuilder.build());
+    assertThat(result.status()).isEqualTo(SEE_OTHER);
+    Optional<Applicant> finalApplicant =
+        repo.lookupApplicant(applicant.get().id).toCompletableFuture().join();
+    assertThat(finalApplicant.get().getApplicantData().getDateOfBirth().get().toString())
+        .isEqualTo("2022-07-10");
```

Comment:
Shouldn't expected DOB be `2022-10-05` as that's what you passed to `updateDateOfBirth` method.

Reply:
No the DB always stores DOB in yyyy-mm-dd format. This is followed everywhere.